api-validation-scoring/about.hbs.md

```diff
@@ -1,5 +1,8 @@
 # Overview of API Validation and Scoring
 
+> **Caution** The APIx package used for API scoring and validation is deprecated.
+> It will be removed in TAP 1.8.
```

Comment:
Is the abbreviation "TAP" used in other parts of the documentation? I think we may want to spell it out as Tanzu Application Platform and then not reference a version that does not exist yet. Maybe say next version of Tanzu Application Platform?